packaging/installer/UPDATE.md

```diff
@@ -117,4 +128,31 @@ sudo ./netdata-installer.sh
 > ⚠️ If you installed Netdata with any optional parameters, such as `--no-updates` to disable automatic updates, and
 > want to retain those settings, you need to set them again during this process.
 
+## Control automatic updates
+
+Starting with Netdata v1.34.0, you can easily enable or disable automatic updates on an existing installation
+using the updater script.
+
+For most installs on Linux, you can enable auto-updates with:
+
+```bash
+/usr/libexec/netdata/netdata-updater.sh --enable-auto-updates
+```
+
+and disable them with:
 
+```bash
+/usr/libexec/netdata/netdata-updater.sh --disable-auto-updates
+```
+
+For static installs, instead use:
+
+```bash
+/opt/netdata//usr/libexec/netdata/netdata-updater.sh --enable-auto-updates
```

Comment:
Just to verify that we're not introducing syntax errors: 
`//` is correct and expected?

Reply:
The `//` was not intentional, but is harmless. POSIX path handling (which Linux follows) mandates that empty path elements inside a path (`//`) are treated as if they do not exist.

I’ll fix this anyway though, as it looks better for us to just list the path properly.